docs/migrating-from-knative-build.md

```diff
@@ -0,0 +1,143 @@
+# Migrating from [Knative Build](https://github.com/knative/build)
+
+This doc describes a process for users who are familiar with Knative `Build` and
+`BuildTemplate` resources to migrate to Tekton `TaskRuns` and `Tasks`, respectively.
+
+Tekton's resources are heavily influenced by Knative's Build-related resources,
+with some additional features that enable them to be chained together inside a
+Pipeline, and provide additional flexibility and reusability.
+
+| **Knative**          | **Tekton**  |
+|----------------------|-------------|
+| Build                | TaskRun     |
+| BuildTemplate        | Task        |
+| ClusterBuildTemplate | ClusterTask |
+
+## Important differences
+
+* All `steps` must have a `name`. Step statuses are not reported in a
```

Comment:
This should be fixed by #963

Reply:
This paragraph needs an update now that #963 is merged 🙏 ❤️

Reply:
Do steps still have to have a name after #963 then?

Reply:
yep they still have a name! i think the comment is about 'step statuses are not reported in the same order' cuz #963 made it so that they are

(i had to re-read this several times to understand which sentence the comment was on haha)